spec.bs

```diff
@@ -422,6 +422,31 @@ action of the user), then return.":
         {{DOMException}}.
     2. <a>Consume user activation</a> given <a>this</a>'s <a>relevant global object</a>.
 
+## Focusing the opener window ## {#focusing-the-opener-window}
+
+<p>
+It can often be useful for the picture-in-picture window to be able to re-focus
+its opener tab, e.g. when the smaller form-factor of the window doesn't fit the
+experience the user needs. We modify the {{Window/focus()}} API to allow it to
+take system-level focus when a picture-in-picture window is focusing its
+opener.
+</p>
+
+<p class="issue">
+Merge this into {{Window/focus()}} once it has enough consensus.
+</p>
+
+Add a new step to {{Window/focus()}} after step 3, "Run the <a>focusing steps</a> with |current|.":
+
+4. If |current| is a <a>top-level traversable</a>, then:
+    1. Let |pipWindow| be |current|'s <a>active window</a>'s
+        <a>documentPictureInPicture API</a>'s <a>last-opened window</a>.
+    2. If |pipWindow| is not <code>null</code> and |pipWindow|'s <a>relevant global object</a>
+        is the <a>incumbent global object</a>, and |pipWindow|'s <a>relevant global object</a>
+        has <a>transient activation</a>, then:
+        1. <a>Consume user activation</a> given |pipWindow|'s <a>relevant global object</a>.
+        2. Give |current| <a>system focus</a>.
```

Comment:
Does this also close the PiP window?

Reply:
No, this will not necessarily close the PiP window. For Chrome specifically, the only time we'd close it is if it was opened via autopip, since we always close autopip when the user focuses the original tab. The website can choose to close the PiP window anytime though via `pipWindow.close()`

Reply:
Done